mockito/invocation.py

```diff
@@ -215,3 +220,15 @@ def __init__(self, return_value):
 
     def answer(self):
         return self.return_value
+
+class ReturnAnswer(object):
+    def __init__(self, mock, answerable):
+        self.answerable = answerable
+        self.mock = mock
+
+    def answer(self):
+        if len(inspect.getargspec(self.answerable).args) == 0:
+            return self.answerable()
+        else:
+            return self.answerable(*self.mock.invocations[0].params)
+
```

Comment:
The `if...then` is not needed b/c if params is empty (the initial value is an empty tuple I think. The `*params` would still work. Support for `kwargs` here would be :sunglasses: